validate_rpm_specs.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
```

Comment:
Question: do we need to add the Copyright statement to *every* file ? Creating a text file with it in the root directory is not enough?